docs/contributing.md

```diff
@@ -1,24 +1,32 @@
 # Contributing to FLORAS
 
-If you want to modify FLORAS or contribute, you can also install it directly from source. We are using pdm to manage the dependencies.
+If you want to modify FLORAS or contribute, you can install it directly from source. We are using pdm to manage the dependencies.
 ```
 pip install pdm
-git clone https://github.com/tulip-control/flowsynth
+git clone https://github.com/tulip-control/floras.git
 ```
-Navigate to the repo and run to install the FLORAS and all required dependencies:
+Navigate to the repo to install floras and all required dependencies:
```

Comment:
The use of upper vs. lower case "FLORAS" and "floras" is somewhat confusing. Tulip is an example of an original name that was notoriously difficult to write: "TuLiP", followed over the years by several variations on this ("tulip", "Tulip", `tulip`). I recommend to decide on a style now and try to be consistent in the documentation.

Reply:
Agreed, floras seems easiest.